text/0000-pack-publish-buildpack.md

```diff
@@ -0,0 +1,276 @@
+# Meta
+[meta]: #meta
+- Name: Pack Publish Buildpack
+- Start Date: 2020-04-27
+- Author(s): [Joe Kutner](https://github.com/jkutner), [Javier Romero](https://github.com/jromero)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: https://github.com/buildpacks/rfcs/blob/master/text/0022-client-side-buildpack-registry.md
+
+# Summary
+[summary]: #summary
+
+This RFC describes the implementation the following commands:
+* `pack publish-buildpack`
+* `pack yank-buildpack`
+
+# Motivation
+[motivation]: #motivation
+
+The [Client-Side Registry RFC (0022)](https://github.com/buildpacks/rfcs/blob/master/text/0022-client-side-buildpack-registry.md) established that two CLI commands would allow users to add/remove buildpacks in the registry, but the implementation of those commands was vauge. In this document we describe the details of how they will work.
+
+# What it is
+[what-it-is]: #what-it-is
+
+## Adding a Buildpack to the Registry
+
+When a buildpack author would like to publish a version of a buildpack on the registry, they will use the `pack publish-buildpack <url>` command. This command will create a Github Issue on the `https://github.com/buildpack/registry` (or similarly named) Github repo requesting the addition the new buildpack version to the index.
```

Comment:
For future proofing and clarity, we should use the repo name "registry-index". "registry" makes me think of the component or service that serves up the index.

```suggestion
When a buildpack author would like to publish a version of a buildpack on the registry, they will use the `pack publish-buildpack <url>` command. This command will create a Github Issue on the `https://github.com/buildpack/registry-index` (or similarly named) Github repo requesting the addition the new buildpack version to the index.
```